store/wrapper_txn.go

```diff
@@ -4,6 +4,16 @@ import (
 	"bytes"
 	"github.com/canopy-network/canopy/lib"
 	"github.com/dgraph-io/badger/v4"
+	"reflect"
+	"unsafe"
+)
+
+const (
+	badgerMetaFieldName               = "meta"    // badgerDB Entry 'meta' field name
+	badgerDiscardEarlierVersions byte = 1 << 2    // badgerDB 'discard earlier versions' flag
+	badgerDeleteBit              byte = 1 << 0    // badgerDB 'tombstoned' flag
+	badgerNoDiscardBit           byte = 1 << 3    // badgerDB 'never discard'  bit
+	badgerGCRatio                     = .00000001 // the ratio when badgerDB will run the garbage collector
```

Comment:
any particular reason of setting this to the lowest possible value? my only concern is that, as the documentation implies, it would increase the activity of the underlying LSM tree thus in the worst case affecting the block commit times on validators which may be setup on slower instances